vip-support/class-vip-support-user.php

```diff
@@ -472,7 +472,7 @@ public function filter_vip_support_email_gravatars( $url, $id_or_email, $args )
 			$user_email = $id_or_email->user_email;
 		}
 
-		if ( isset( $user_email ) && self::is_a8c_email( $user_email ) && ( ! $user || $this->has_vip_support_meta( $user->ID ) ) ) {
+		if ( isset( $user_email ) && self::is_a8c_email( $user_email ) && ( isset( $user->ID ) && $this->has_vip_support_meta( $user->ID ) ) ) {
```

Comment:
Was there a bug in the original condition? I'm also not clear on the intent of this check.

Reply:
Yep, looks like there was. It checks if the user has an a8c email and also checks if it has the meta attached to it. In an odd case where `$user` isn't defined, we should check if it's set.